caikit/runtime/service_factory.py

```diff
@@ -115,6 +119,7 @@ def get_service_package(
                 concrete Servicer implementation to a protobufs Service and grpc Server
         """
         if service_type == cls.ServiceType.TRAINING_MANAGEMENT:
+            # TODO: should this use service_generation.get_runtime_service_package()?
```

Comment:
We're breaking the API, so yes, we should NOT be hard-coding this for the management service